src/Stratis.Features.SQLiteWalletRepository/SQLiteWalletRepository.cs

```diff
@@ -1574,11 +1574,23 @@ public IEnumerable<TransactionData> GetTransactionOutputs(HdAccount hdAccount, D
 
                 if (!addressDict.TryGetValue(addressIdentifier, out HdAddress hdAddress))
                 {
-                    ExtPubKey extPubKey = ExtPubKey.Parse(hdAccount.ExtendedPubKey, this.Network);
+                    string pubKeyHex = null;
```

Comment:
Looks good. In fact, the code added for watch-only-wallets may work equally well for the general case.